var/spack/repos/builtin/packages/cctools/package.py

```diff
@@ -24,7 +25,6 @@ class Cctools(AutotoolsPackage):
     version('6.1.1', sha256='97f073350c970d6157f80891b3bf6d4f3eedb5f031fea386dc33e22f22b8af9d')
 
     depends_on('openssl')
-    depends_on('perl+shared', type=('build', 'run'))
```

Comment:
Can you say a bit more why `perl` was removed as a dependency?

Reply:
Massimiliano, indeed, it is a mistake to remove the dependency.